### ISSUE TITLE
A radical solution to all problems

### ISSUE THREAD

**Issue by d3nchk** [hzqst/MetaHookSv#219]
Good evening!
I had a genius idea:
Maybe fuck OpenGL, can you move everything to Vulkan? The drivers eat it fine, works equally well on AMD and NVIDIA. I just don't think AMD will ever solve the OpenGL problem.
And the performance on AMD is at least better on Vulkan and reaches 300% difference in favor of Vulkan.
At the moment the game keeps crashing after 1-2 maps and there's nothing you can do about it. It's 100% AMD drivers.

**Comment by hzqst** [comment]
Good idea but not practical for mods like Sven Co-op as some elements in such mods are rendered in raw OpenGL call instead of engine efx API or vgui2 Surface API, which makes it almost impossible to redirect those raw OpenGL call to another API unless there is a abstract layer (e.g. DXVK) that translates raw OpenGL call to Vulkan call.
